src/NewRelic.Telemetry/Extensions/DateTimeExtensions.cs

```diff
@@ -24,7 +24,14 @@ internal static class DateTimeExtensions
         private const long UnixEpochTicks = DaysTo1970 * TicksPerDay;
         private const long UnixEpochMilliseconds = UnixEpochTicks / TicksPerMillisecond; // 62,135,596,800,000
 
-        internal static long ToUnixTimeMilliseconds(this DateTimeOffset dateTime)
+        private static readonly DateTime Epoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        public static long ToUnixTimeMilliseconds(this DateTimeOffset dateTime)
+        {
+            return (dateTime.ToUniversalTime().Ticks / TicksPerMillisecond) - UnixEpochMilliseconds;
+        }
+
+        public static long ToUnixTimeMilliseconds(this DateTime dateTime)
```

Comment:
We added an overload that takes DateTime? How come?

Reply:
removed